server/pkg/integrationapi/event_test.go

```diff
@@ -0,0 +1,174 @@
+package integrationapi
+
+import (
+	"testing"
+	"time"
+
+	"github.com/reearth/reearth-cms/server/pkg/asset"
+	"github.com/reearth/reearth-cms/server/pkg/event"
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearth-cms/server/pkg/operator"
+	"github.com/reearth/reearth-cms/server/pkg/project"
+	"github.com/reearth/reearthx/account/accountdomain"
+	"github.com/reearth/reearthx/account/accountdomain/user"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_NewOperator(t *testing.T) {
+
+	uid := accountdomain.NewUserID()
+	integrationID := id.NewIntegrationID()
+	opUser := operator.OperatorFromUser(uid)
+	opIntegration := operator.OperatorFromIntegration(integrationID)
+	opMachine := operator.OperatorFromMachine()
+	tests := []struct {
+		name  string
+		input operator.Operator
+		want  Operator
+	}{
+		{
+			name:  "success user operator",
+			input: opUser,
+			want: Operator{
+				User: &OperatorUser{
+					ID: uid.String(),
+				},
+			},
+		},
+		{
+			name:  "success integration operator",
+			input: opIntegration,
+			want: Operator{
+				Integration: &OperatorIntegration{
+					ID: integrationID.String(),
+				},
+			},
+		},
+		{
+			name:  "success machine operator",
+			input: opMachine,
+			want: Operator{
+				Machine: &OperatorMachine{},
+			},
+		},
+		{
+			name:  "success unknown operator",
+			input: operator.Operator{},
+			want:  Operator{},
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
+			result := NewOperator(test.input)
+			if !assert.Equal(t, result, test.want) {
+				t.Errorf("expected %+v but got %+v", test.want, result)
+			}
```

Comment:
```suggestion
			assert.Equal(t, result, test.want)
```
no need to call r.Errorf method, assert.Equal will print the error if any